docs/backups.MD

```diff
@@ -0,0 +1,115 @@
+## VMBackuper
+
+#### vmbackuper is proprietary software. 
+
+ `VMSingle` and `VMCluster` has built-in backup configuration, it uses `vmbackuper` - proprietary tool for backups.
+ It supports incremental backups (hours, daily, etc) with popular object storages (aws s3, google cloud storage).
+ 
+  You can enable it with the simple configuration, define secret
+  
+  ```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: remote-storage-keys
+type: Opaque
+stringData:
+  credentials: |-
+    [default]
+    aws_access_key_id = your_access_key_id
+    aws_secret_access_key = your_secret_access_key
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMSingle
+metadata:
+  name: example-vmsingle
+spec:
+  # Add fields here
+  retentionPeriod: "1"
+  vmBackup:
+    destintaion: "s3://your_bucket/folder"
+    credentialsSecret:
+      name: remote-storage-keys
+      key: credentials
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMCluster
+metadata:
+  name: example-vmcluster-persistent
+spec:
+  retentionPeriod: "4"
+  replicationFactor: 2
+  vmstorage:
+    replicaCount: 2
+    vmBackup:
+      destintaion: "s3://your_bucket/folder"
```

Comment:
same here.

Reply:
There was a problem with CRD tag naming, fixed it. Thanks!